packages/generator-single-spa/src/root-config/templates/src/activity-functions.js

```diff
@@ -1,4 +1,3 @@
-export function navbar(location) {
-  // The navbar is always active
+export function always(location) {
```

Comment:
This seemed more generic and easier to understand (for me).

Reply:
Can't this entire file be deleted now that we're using the object version of registerApplication?

Reply:
I like it!